.gitignore

```diff
@@ -76,13 +76,15 @@ lms/static/certificates/css/
 cms/static/css/
 
 ### Styling generated from templates
-lms/static/sass/lms-main.scss
-lms/static/sass/lms-main-rtl.scss
-lms/static/sass/lms-course.scss
-lms/static/sass/lms-course-rtl.scss
-lms/static/sass/lms-footer.scss
-lms/static/sass/lms-footer-rtl.scss
-
+lms/static/sass/*.css
```

Comment:
Nice general clean up here! Good to see whole dirs being covered rather than individual Sass compilation files.